featuretools/computational_backends/calculate_feature_matrix.py

```diff
@@ -136,10 +136,25 @@ def calculate_feature_matrix(features, entityset=None, cutoff_time=None, instanc
             # take the first column that isn't instance_id and assume it is time
             not_instance_id = [c for c in cutoff_time.columns if c != "instance_id"]
             cutoff_time.rename(columns={not_instance_id[0]: "time"}, inplace=True)
+        if cutoff_time['time'].dtype == object:
+            if entityset.time_type == NumericTimeIndex:
+                try:
+                    cutoff_time['time'] = pd.to_numeric(cutoff_time['time'])
+                except (ValueError, TypeError):
+                    raise TypeError("cutoff_time times must be a numeric")
+            else:
+                try:
+                    cutoff_time['time'] = pd.to_datetime(cutoff_time['time'])
+                except (ValueError, TypeError):
+                    raise TypeError("cutoff_time times must be datetime type")
         pass_columns = [column_name for column_name in cutoff_time.columns[2:]]
 
     if _check_time_type(cutoff_time['time'].iloc[0]) is None:
         raise ValueError("cutoff_time time values must be datetime or numeric")
+    if cutoff_time['instance_id'].dtype == np.dtype('O') and target_entity.df[target_entity.index].dtype.name.find('int') > -1:
```

Comment:
I think we can do without these checks and conversations for now. let's assume the user has this fixed up before calling calculate feature matrix.

Reply:
These are to suppress warnings from the tests. Should we change the tests then? Or add in a filter to ignore pandas warnings when running the tests?